docs/source/release_notes/releases.rst

```diff
@@ -40,61 +40,53 @@ v4.1.1 (2020-10-29)
 Features
 ~~~~~~~~
 
-- GitHub Action to ensure that each pull request into main makes an associated release note entry. (`#2396 <https://github.com/nucypher/nucypher/issues/2396>`__)
-
-
-v4.1.0 (2020-10-19)
--------------------
-
-No significant changes.
-
-
-v4.0.1 (2020-10-14)
--------------------
-
-No significant changes.
-
+- Add CLI functionality for the removal of unused sub-stakes to save Ursula gas costs. (`#2384 <https://github.com/nucypher/nucypher/issues/2384>`__)
 
-v4.0.0 (2020-10-14)
--------------------
-
-No significant changes.
 
+Bugfixes
+~~~~~~~~
 
-v3.0.0-beta.4 (2020-10-12)
---------------------------
+- Automatically restart Ursula worker task on failure. (`#2410 <https://github.com/nucypher/nucypher/issues/2410>`__)
 
-No significant changes.
 
+Improved Documentation
+~~~~~~~~~~~~~~~~~~~~~~
 
-v3.0.0-beta.3 (2020-09-30)
---------------------------
+- Update global fee range documentation, including genesis values. (`#2363 <https://github.com/nucypher/nucypher/issues/2363>`__)
 
-No significant changes.
 
+Misc
+~~~~
 
-v3.0.0-beta.2 (2020-09-25)
---------------------------
+- Update Ursula network grant availability script for mainnet usage. (`#2383 <https://github.com/nucypher/nucypher/issues/2383>`__)
+- GitHub Action to ensure that each pull request into main makes an associated release note entry. (`#2396 <https://github.com/nucypher/nucypher/issues/2396>`__)
 
-No significant changes.
 
+v4.1.0 (2020-10-19)
+-------------------
 
```

Comment:
```suggestion

Features
~~~~~~

- Reworked design for the status page. (`#2351 <https://github.com/nucypher/nucypher/pull/2351>`__)

```

Reply:
Unfortunately, PR #2351 was introduced in 4.0.0 (which we just have as `Mainnet Launch`) which is why I didn't include it.

Reply:
Oh yes, you're right, I looked at a wrong merge. Sorry about that.